modules/develop/pages/produce-data/leader-pinning.adoc

```diff
@@ -0,0 +1,51 @@
+= Leader Pinning
+:page-categories: Clients, Development
+// tag::single-source[]
+:description: Learn about leader pinning and how to configure a preferred partition leader location based on cloud availability zones or regions.
+
+Produce requests to write data to Redpanda topics go through the topic partition leader, which syncs messages across its follower replicas. For a Redpanda cluster deployed across multiple availability zones (AZs), leader pinning ensures that a topic's partition leaders are geographically closer to clients. Leader pinning helps decrease networking costs and guarantee lower latency.
+
+xref:develop:consume-data/follower-fetching.adoc[Follower fetching], where you configure consumers to fetch records from the closest replica, can also help reduce overall networking costs. If consumers are located in the same preferred region or AZ for leader pinning, and you have not enabled follower fetching, you still see additional benefits for consume requests by enabling leader pinning.
```

Comment:
I'm not sure about the phrasing here...

in the same preferred region or AZ _for_ leader pinning

do you mean as?